### ISSUE TITLE
Azure P2S VPN Gateway Tunnel to Multiple to Azure Subscriptions

### ISSUE THREAD

**Issue by prasadaws10** [RY3IEZN/Azure_p2s#1]
Is it possible to use Azure Point to Site VPN tunnel with VPN gateway to connect with multiple azure subscriptions like dev and prod. But no other resources/data sharing between dev and prod subscriptions

**Comment by RY3IEZN** [comment]
Yes, You can achieve that by putting them in separate Vnets

**Comment by prasadaws10** [comment]
You mean like having VPN Gateway in one separate Azure subscription Subnet/vnet and peer that to other Dev/Prod/Staging Vnets in other Dev/Prod/Staging Subscriptions ? So then the peered vnets will have access to only those connections ?

**Comment by prasadaws10** [comment]
Yes. the goal is to keep each resources/data separately from dev vs prod vs staging. Is this a feasible solution to have VPN gateway in one separate Azure subscription and peer that vpn gateway's vnet to other dev/stg/prod subscription vnets ? Is it a proper solution?

Actually, I want to access Azure through a secured route/VPN tunnel, as I don't have any on-prem to set up any site-to-site/express route pattern, so I opted to point to site solution using a VPN gateway.

But I am having three subscriptions all total to access, rather having three different VPN gateways in each subscription, I am thinking the above solution.

Thank you.

**Comment by RY3IEZN** [comment]
Thanks for a better explanation, yes you can have 1 vpn gateway and 3 different vnets... 

But im not sure the vpn can be connected to 3 vnets at the same time. You may have to manually switch or should i say disconnect and reconnect between vnets.. which can be tedious/mistakes can happen
